actions/channel_actions.jsx

```diff
@@ -75,7 +75,7 @@ export function loadChannelsForCurrentUser() {
         const state = getState();
         const unreads = getUnreadChannelIds(state);
 
-        await dispatch(ChannelActions.fetchMyChannelsAndMembers(getCurrentTeamId(state)));
+        await dispatch(ChannelActions.fetchMyChannelsAndMembersREST(getCurrentTeamId(state)));
```

Comment:
Just a bunch of places where we append REST for actions, which we are supporting in GRAPHQL

Reply:
Like the clarity and convention 👍 it will be easy to delete them if needed in the future.